openapi/components/schemas/RunRequest.yaml

```diff
@@ -47,3 +47,6 @@ description: >-
   To execute a workflow, send a run request including all the details needed to begin downloading
 
   and executing a given workflow.
+  
+  If workflow_engine and workflow_engine_version is not provided, clients can use the most recent workflow_engine_version of workflow_enginer that WES instance uses to process the request if 
```

Comment:
Do you mean `servers` or `implementers` rather than `clients`?

Also, typo in second mention of `workflow_engine` (trailing `r`).

Reply:
Fixed